src/lib/Scenes/Artwork/Components/AboutArtist.tsx

```diff
@@ -27,7 +27,9 @@ export class AboutArtist extends React.Component<AboutArtistProps> {
             {hasSingleArtist ? "About the artist" : "About the artists"}
           </Sans>
           {artists.map(artist => (
-            <ArtistListItem key={artist.id} artist={artist} />
+            <Box width="100%" py={1} key={artist.id}>
```

Comment:
cc @kierangillen this should be good for another review.

I noticed that the other usages of the `ArtistListItem` added their own padding, so I only stuck the custom behavior here.

Reply:
Hey would it be possible to add an image to the PR showing this component didn't visually change?

Reply:
The `<ArtistListItem>` didn't have any internal padding previously so we shouldn't need to add a wrapper.

Reply:
Not sure I understand-- this wrapper is so that when there are multiple, the components have some padding between them.

Reply:
![image](https://user-images.githubusercontent.com/2081340/60621564-feb8c080-9dab-11e9-8d8f-049da7719a75.png)

Reply:
Oh oh, you're referring to whether or not we need the `Box`-- let me check.